modules/apps/object/object-test/src/testFunctional/tests/ObjectExportImport.testcase

```diff
@@ -179,13 +179,7 @@ definition {
 		}
 
 		task ("And given the object definition is imported") {
-			ObjectExportImport.importObject(
-				objectFileName = "ImportedObjectWith100Fields",
-				objectName = "CustomObject187390");
-
-			ObjectPortlet.selectCustomObject(label = "Custom Object 187390");
-
-			ObjectAdmin.publishObjectViaUI();
+			ObjectExportImport.importExportedObject(objectName = "CustomObject187390");
```

Comment:
This new macro imports the file directly from the temporary folder (where it is after download). In other words, we are importing exactly the same object that was exported, avoiding using a file from the Dependencies folder.